library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -3078,6 +3078,8 @@ void libspdm_reset_context(void *spdm_context)
     context->mut_auth_cert_chain_buffer_size = 0;
     context->current_dhe_session_count = 0;
     context->current_psk_session_count = 0;
+
+    context->local_context.cert_slot_reset_mask = 0;
```

Comment:
We should not reset cert_slot_reset_mask as you mentioned " reset mask should be sticky between GET_VERSIONS"?

Reply:
Ah yeah forgot to delete that. Thanks.